src/Symfony/Bundle/FrameworkBundle/Controller/ControllerTrait.php

```diff
@@ -383,4 +383,22 @@ protected function isCsrfTokenValid(string $id, string $token): bool
 
         return $this->container->get('security.csrf.token_manager')->isTokenValid(new CsrfToken($id, $token));
     }
+
+    /**
+     * Dispatches a message to the bus.
+     *
+     * @param object $message The message to dispatch
+     *
+     * @return mixed The result from the bus
+     *
+     * @final
+     */
+    protected function dispatch($message)
```

Comment:
`dispatchMessage()`? to not cause any confusion with event dispatching.

Reply:
I agree with @ro0NL. `dispatch` is too generic and can also mean sending something via the event dispatcher.

Reply:
Let's go for `dispatchMessage` then